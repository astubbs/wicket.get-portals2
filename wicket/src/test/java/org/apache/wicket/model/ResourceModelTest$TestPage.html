<html>
<body>
<span wicket:id="testlabel">[a test]</span>
</body>
</html>