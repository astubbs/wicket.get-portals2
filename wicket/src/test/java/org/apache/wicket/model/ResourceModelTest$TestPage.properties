test.label=A test label from a resource bundle
