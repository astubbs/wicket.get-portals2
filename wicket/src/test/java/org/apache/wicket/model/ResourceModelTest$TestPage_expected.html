<html>
<body>
<span wicket:id="testlabel">A test label from a resource bundle</span>
</body>
</html>